omise/omise-php
# Add expire method for Charge

### Summary
- [x] Add expire method for Charge ([Omise doc](https://www.omise.co/charges-api#expire))

### QA
You should be able to set a charge to expire.

1. Retrieve a  charge
`$charge = OmiseCharge::retrieve('chrg_test_5mwjsrxkxzufc0wrtfz');`

2. Set a charge to expire
`$charge->expire();`

3. Check whether the charge has been set to expire
`$charge['expired'];`
